src/crypto/error.ts

```diff
@@ -0,0 +1,10 @@
+export class InvalidSeedError extends Error {
+  public static INVALID_SEED_MESSAGE = "Provided seed string needs to be longer or equal than 32 characters";
+  constructor() {
+    super();
+    this.name = "InvalidSeedError";
+    this.message = InvalidSeedError.INVALID_SEED_MESSAGE;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any,@typescript-eslint/consistent-type-assertions
+    this.stack = (<any> new Error()).stack;
+  }
+}
```

Comment:
https://github.com/MetaMask/eth-json-rpc-errors

Reply:
@mpetrunic I will be on the lookout for _typescript migration PR_ to finish and then migrate our errors to use this.

Reply:
it already has typescript definitions

Reply:
but now that im thinking of it, it might make more sense to throw substrate rpc errors